x/genutil/collect.go

```diff
@@ -99,15 +98,18 @@ func CollectTxs(cdc codec.JSONMarshaler, txJSONDecoder sdk.TxDecoder, moniker, g
 	var addressesIPs []string
 
 	for _, fo := range fos {
```

Comment:
IMHO I would prefer the walkdir approach.

Reply:
That approach fully traverses the directory while this one is the correct
equivalent of only traversing the top level directory.